cfn-resources/project/cmd/resource/resource.go

```diff
@@ -559,3 +523,30 @@ func getChangeInAPIKeys(currentKeys []ProjectApiKey, previousKeys []ProjectApiKe
 
 	return newKeys, changedKeys, removeKeys
 }
+
+func changeProjectTeams(atlasV2 admin.APIClient, currentModel *Model, newTeams []admin.TeamRole) (errorMessage string, err error) {
+	newTeams, changedTeams, removeTeams := getChangeInTeams(currentModel.ProjectTeams, newTeams)
+	projectID := *currentModel.Id
+	// Remove Teams
+	for _, team := range removeTeams {
+		_, err = atlasV2.TeamsApi.RemoveProjectTeam(context.Background(), projectID, util.SafeString(team.TeamId)).Execute()
+		if err != nil {
+			return "Error while deleting team from project", err
+		}
+	}
+	// Add Teams
+	if len(newTeams) > 0 {
+		_, _, err = atlasV2.TeamsApi.AddAllTeamsToProject(context.Background(), projectID, &newTeams).Execute()
+		if err != nil {
+			return "Error while adding team to project", err
+		}
+	}
+	// Update Teams
```

Comment:
knit: not sure if these comments are needed or the code is clear enough (or could be made more clear so we don't need comments)